src/k8s-engine/lima.ts

```diff
@@ -1594,6 +1596,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
           this.progressTracker.action('Installing image scanner', 50, this.installTrivy()),
           this.progressTracker.action('Installing CA certificates', 50, this.installCACerts()),
         ]);
+        await this.progressTracker.action('Installing credential helper', 50, this.installCredentialHelper());
```

Comment:
Could this not run in parallel with some of the other tasks?

Reply:
Sure.